guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -106,6 +106,11 @@ ifdef::katello,orcharhino,satellite[]
 include::common/modules/proc_tuning-with-predefined-profiles.adoc[leveloffset=+3]
 endif::[]
 
+ifdef::satellite[]
+//SmartProxy Sync Deadlock Warning
+include::common/modules/snip_deadlock_warning.adoc[leveloffset=+3]
+endif::[]
+
```

Comment:
You're including this as a snippet in master.adoc. Does this mean it's not part of proc_tuning-with-predefined-profiles.adoc? Because right now, it looks like users are supposed to run the commands as the last step of the "Tuning Satellite Server with Predefined Profiles" procedure. But that seems unlikely to me.

When you write "To avoid this, ensure that no Capsule synchronization tasks are in progress", when exactly should users ensure this? Is there an existing procedure into which you could add this as a regular procedure step? For information like this, placement is crucial -- we should make sure users read this advice exactly at the time when they need it.

Reply:
So the procedure needs to be performed post-upgrade, that is the only restriction. Since, the subsections in post-upgrade are optional depending on the setup the user has, I have moved the warning to the top. Does this work?

Reply:
Yes, this works. (Normally, I would consider adding this as [a troubleshooting section](https://redhat-documentation.github.io/modular-docs/#troubleshooting) to an existing procedure. But I can see that this problem exists only in the earliest supported version... so let's skip the trouble of major-ish restructuring :)) Thanks!